Subject: [PATCH] remover Registrar
---
Index: android/src/main/java/dev/legoffmael/imagecrop/ImageCropPlugin.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/android/src/main/java/dev/legoffmael/imagecrop/ImageCropPlugin.java b/android/src/main/java/dev/legoffmael/imagecrop/ImageCropPlugin.java
--- a/android/src/main/java/dev/legoffmael/imagecrop/ImageCropPlugin.java	(revision f2642ac8b9b3412b59f9ff8eaa6f8cfc1cf2fdc5)
+++ b/android/src/main/java/dev/legoffmael/imagecrop/ImageCropPlugin.java	(revision 46dbe891c2773c45002bf1ef72d66e62b3b1f144)
@@ -33,7 +33,6 @@
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
 import io.flutter.plugin.common.MethodChannel.Result;
 import io.flutter.plugin.common.PluginRegistry;
-import io.flutter.plugin.common.PluginRegistry.Registrar;
 import io.flutter.embedding.engine.plugins.FlutterPlugin;
 import io.flutter.embedding.engine.plugins.activity.ActivityAware;
 import io.flutter.embedding.engine.plugins.activity.ActivityPluginBinding;
@@ -57,15 +56,6 @@
 
     public ImageCropPlugin(){ }
 
-    /**
-     * legacy APIs
-     */
-    public static void registerWith(Registrar registrar) {
-        ImageCropPlugin instance = new ImageCropPlugin(registrar.activity());
-        instance.setup(registrar.messenger());
-        registrar.addRequestPermissionsResultListener(instance);
-    }
-
     @Override
     public void onAttachedToEngine(@NonNull FlutterPluginBinding binding) {
       this.setup(binding.getBinaryMessenger());
